crates/client-api/Cargo.toml

```diff
@@ -47,7 +47,7 @@ bytestring = "1"
 tokio-tungstenite.workspace = true
 itoa.workspace = true
 derive_more = "0.99.17"
-uuid.workspace = true
+uuid = { workspace = true, features = ["v4"] }
```

Comment:
Is this necessary? There don't seem to be any other changes in the `client-api` crate that would require this.